src/tests/functional/disk_cache_tests.ts

```diff
@@ -0,0 +1,58 @@
+import { Page, test, expect } from "@playwright/test"
+import { nextBody } from "../helpers/page"
+
+const path = "/src/tests/fixtures/disk_cache.html"
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(path)
+})
+
+test("test stores pages in the disk cache", async ({ page }) => {
```

Comment:
I think this does verify that the cache gets populated, but is it also worth testing that those cached entries definitely get used? Maybe the test could listen for the `turbo:render` event for the cached version when following a link?

Reply:
As discussed offline, it'd be nice to have a test checking the cache (we don't have any at the moment), but for some reason this doesn't work in the test env. Will have to punt this for now.